pages/about.html

```diff
@@ -4,9 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Awesome Sunny</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header> 
+    <h1 class="page-titles">About Sunny</h1>
+    <nav>
+      <ul class="container">
+        <li>
+          <a href = "portfolio.html">Portfolio</a> 
```

Comment:
It's hard to see on the diff in github, but this line and a few others have extra trailing spaces, which my HTML validator did catch. (I'll skip pointing out the other times.)

A very minor style issue, but admittedly with more weight on the web as each space is another byte...